discord_bot.js

```diff
@@ -158,16 +158,19 @@ commands = {
       }
     },
   },
-  idle: {
-    description: "Sets bot status to idle.",
+  status: {
+    description: "Sets bot status to idle/online.",
     process: function (bot, msg, suffix) {
-      bot.user.setStatus("idle").then(console.log).catch(console.error);
-    },
-  },
-  online: {
-    description: "Sets bot status to online.",
-    process: function (bot, msg, suffix) {
-      bot.user.setStatus("online").then(console.log).catch(console.error);
+      if(massage.author.id !== yor id here) return; // here you check if the message auhor id was the same (so only you can use it)
```

Comment:
I would probably remove this line and let people restrict command access via the permission system. Otherwise the bot might break if someone doesn't manually edit this line.

Reply:
yeah

Reply:
+1 on using the permission system to restrict access. Could make it a default restricted command if we're worried about it, although changing the bot's status is relatively harmless.